server/src/main/kotlin/fi/oph/kitu/yki/YkiViewController.kt

```diff
@@ -21,12 +22,20 @@ class YkiViewController(
     fun suorituksetView(
         model: Model,
         response: HttpServletResponse?,
+        @RequestParam("versionHistory") versionHistory: Boolean?,
     ): ModelAndView {
-        val suoritukset: List<YkiSuoritusEntity> = suoritusRepository.findAll().toList()
+        val suoritukset: List<YkiSuoritusEntity> =
+            if (versionHistory != null && versionHistory) {
+                suoritusRepository.findAll().toList()
```

Comment:
Pitäisiköhän samaa logiikkaa käyttää CSV:n exporttauksessa? Kun käyttäjä lataa CSV:n? Mietin vaan et jos pidettäis näitä kahta logiikkaa yhtenäisinä.

Reply:
Mietin tätä kans, ja nyt kun katon niin eihän se mikään suuri homma näyttäisi olevan 🤔 Ehkä lisään sen viel tähän pullariin